analysis/import/SonarImporter/build.gradle

```diff
@@ -12,13 +12,15 @@ dependencies {
     runtime group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: jersey_version
     runtime group: 'com.sun.activation', name: 'javax.activation', version: '1.2.0'
 
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: junit5_version
-testImplementation group: 'org.hamcrest', name: 'hamcrest-core', version: hamcrest_version
+    testImplementation group: 'junit', name: 'junit', version: junit4_version
```

Comment:
Any reason to fall back to the older version?

Reply:
all tests were written in junit4. junit5 was never used